materialize-bigquery/sqlgen.go

```diff
@@ -122,23 +122,6 @@ CLUSTER BY {{ range $ind, $key := $.Keys }}
 	{{- end}};
 {{ end }}
 
--- Alter column and mark it as nullable
-
-{{ define "alterColumnNullable" }}
-ALTER TABLE {{ $.Table.Identifier }} ALTER COLUMN {{ $.Identifier }} DROP NOT NULL;
```

Comment:
Previously this would error if the column did not have a `NOT NULL` constraint, which made it effectively impossible to remove a non-required field from a materialized collection.